extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,10 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery {
+  display: none;
+  z-index: 2147483645 !important;
+  top: 50px !important;
+  overflow: auto !important;
+}
+
+.-amp-lightbox-viewer[gallery-view] .-amp-lightbox-viewer-gallery {
+  display: block !important;
+}
+
+.-amp-lightbox-viewer-gallery-thumbnail {
+  display: inline-block;
+  object-fit: contain;
+  width: calc(100vw/3);
+  height: calc(100vw/3);
+  padding: 5px;
+  box-sizing: border-box;
+  overflow: hidden;
+}
+
+.-amp-lightbox-viewer-gallery-thumbnail-img {
+  width: 100%;
```

Comment:
there's no thumbnail element on the example file, we will create an image and append to page now.